Scripts/extract_release_notes.rb

```diff
@@ -0,0 +1,81 @@
+# Parses the release notes file to extract the current version information and
+# puts it to STDOUT.
+#
+# To update the release notes for localization:
+#
+# ruby ./this_script >| Simplenote/Resources/release_notes.txt
+#
+# To generate the App Store Connect release message:
+#
+# ruby ./this_script | pbcopy
+#
+# To generate the GitHub and App Center release message:
+#
+# ruby ./this_script -k | pbcopy
+
+GITHUB_URL = 'https://github.com/Automattic/simplenote-ios'
+
+RELEASE_NOTES_FILE = 'RELEASE-NOTES.txt'
+NOTES = File.read(RELEASE_NOTES_FILE)
```

Comment:
Don't we have another `.txt` file that have the release notes just for the current version (already extracted and isolated) by one of our release-toolkit lanes/actions already (somewhere in `fastlane/metadata/`)?

Parsing that other file instead of the `RELEASE-NOTES.txt` would allow you to simplify that script a lot by avoid to have to do the whole "ignore the first empty section for next version" + "ignore the current version header" + "parse up until next empty line" (as that would have already been done by our release-toolkit step), and make this script only have to do the `gsub(/^- */,'- ')` part + the `replace_pr_number_with_markdown_link` call?